src/extractors/fontawesome6extractor.py

```diff
@@ -22,20 +22,28 @@ def __fetch_icons(self: 'FontAwesome6Extractor'):
         for (name, data) in response.json().items():
             icon = int(data['unicode'], 16)
             name = f'fa-{name}'
-            self.__icons.append(Character(icon, name))
+            char = Character(icon, name)
+            char.aliases: List[str] = []
             try:
                 for alias in data['aliases']['names']:
-                    self.__icons.append(Character(icon, f'fa-{alias}'))
+                    char.aliases.append(f'fa-{alias}')
             except KeyError:
                 pass
+            self.__icons.append(char)
 
     def __write_to_file(self, target: Path):
         if len(self.__icons) == 0:
             return
 
         with (target / 'fontawesome6.csv').open('w') as symbol_file:
             for icon in self.__icons:
-                symbol_file.write(f'{icon.directional_char} {html.escape(icon.name.lower())}\n')
+                line=f'{icon.directional_char} {html.escape(icon.name.lower())}'
+                if len(icon.aliases) > 0:
+                    line += ' <small>(' + ', '.join(icon.aliases) + ')</small>\n'
```

Comment:
You could probably make it easier with `f" <small>{', '.join(icon.aliases)}</small>\n"`, if you want